### ISSUE TITLE
2.5 backport PR request list

### ISSUE THREAD

**Issue by ManfeiBai** [pytorch/xla#7977]
Similar to the backport request thread in 2.2 release https://github.com/pytorch/xla/issues/6036

The issue is to track 2.5 release backport.

For any PRs you want to backport to 2.5, please reply with following:

Original PR link
Reason to backport
2.5 backport PR link

**Comment by zpcore** [comment]
PR: https://github.com/pytorch/xla/pull/8003

Reason: Correct the warning message for 2.5 release.

Cherrypick: No done.

**Comment by ManfeiBai** [comment]
PR: #7916

Reason: OpenXLA-pin update to Sep13

Cherrypick: WIP

**Comment by ManfeiBai** [comment]
PR: https://github.com/pytorch/xla/pull/8008

Reason: pallas test fix for OpenXLA-pin update to Sep13

Cherry-pick: WIP

**Comment by jeffhataws** [comment]
PR: https://github.com/pytorch/xla/pull/7992

Reason: Fix for autocast to enable cross entropy loss with FP32 precision

Cherry-pick: to be done

**Comment by bhavya01** [comment]
PR: https://github.com/pytorch/xla/pull/8202

Reason: Fix dataloading issue for SD2 model

Cherry-pick: WIP

**Comment by jeffhataws** [comment]
PR: https://github.com/pytorch/xla/pull/8204
Reason: Multi-node SPMD support for Neuron
Cherry-pick: WIP

**Comment by ManfeiBai** [comment]
Thanks, since 2.5 has been release, will close this issue
